tasks/build_tags.py

```diff
@@ -109,19 +111,19 @@
 FIPS_AGENT_TAGS = AGENT_TAGS.union({"goexperiment.systemcrypto"})
 
 # CLUSTER_AGENT_TAGS lists the tags needed when building the cluster-agent
-CLUSTER_AGENT_TAGS = {"clusterchecks", "datadog.no_waf", "kubeapiserver", "orchestrator", "zlib", "zstd", "ec2"}
+CLUSTER_AGENT_TAGS = {"clusterchecks", "datadog.no_waf", "glibc", "kubeapiserver", "orchestrator", "zlib", "zstd", "ec2"}
```

Comment:
I ran your branch through our internal CI and this came up on the python linter:
```suggestion
CLUSTER_AGENT_TAGS = {
    "clusterchecks",
    "datadog.no_waf",
    "glibc",
    "kubeapiserver",
    "orchestrator",
    "zlib",
    "zstd",
    "ec2",
}
```